src/PackagePublisher/PackagePublisher.csproj

```diff
@@ -1,13 +1,37 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\build\common.props" />
+  
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <Description>Publishes packages to a feed, with retries.</Description>
+    <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
Can we use the same version as everything else in this repo (assuming they are all have the same version)?